src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ClientCreds/JsonWebToken.cs

```diff
@@ -225,6 +228,18 @@ public string X509CertificateThumbprint
 
                 set { _thumbPrint = value; }
             }
+
+            [DataMember(Name = JsonWebTokenConstants.ReservedHeaderParameters.X509Certificate)]
+            public string X509Certificate
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this an auto-implemented property and remove its backing field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS2292)